apps/web/screens/Applications/Applications.tsx

```diff
@@ -0,0 +1,331 @@
+                          <Link
+                            {...linkResolver('article', [article.slug])}
+                            skipTab
+                          >
+                            <Button
+                              variant="text"
+                              size="small"
+                              icon="arrowForward"
+                              nowrap
+                            >
+                              {gn('readMore', 'Sjá nánar')}
+                            </Button>
+                          </Link>
+                        </T.Data>
+                      </T.Row>
+                    ))}
+                  </T.Body>
+                </T.Table>
+              )}
+            </Box>
+            {totalSearchResults > 0 && (
+              <Box paddingTop={6}>
+                <Pagination
+                  page={page ?? 1}
+                  totalPages={totalPages}
+                  variant="blue"
+                  renderLink={(page, className, children) => (
+                    <Link
+                      href={{
+                        pathname: linkResolver('applications').href,
+                        query: { ...query, page },
+                      }}
+                    >
+                      <span className={className}>{children}</span>
+                    </Link>
+                  )}
+                />
+              </Box>
+            )}
+          </GridColumn>
+        </GridRow>
+      </GridContainer>
+    </>
+  )
+}
+
+Applications.getProps = async ({ apolloClient, locale, query }) => {
+  // TODO - file bugreport with next-usequerystate regarding missing default on server
+  const {
+    q = '*',
+    page: pageParam = '1',
+    category: categoryParam,
+    organization: organizationParam,
+  } = query
+  const queryString = parseAsString.withDefault('*').parseServerSide(q)
+  const page = parseAsInteger.withDefault(1).parseServerSide(pageParam)
+  const order = SortDirection.Desc
+  const category = parseAsArrayOf(parseAsString)
+    .withDefault([])
+    .parseServerSide(categoryParam)
+  const organization = parseAsArrayOf(parseAsString)
+    .withDefault([])
+    .parseServerSide(organizationParam)
+  const types = [SearchableContentTypes['WebArticle']]
+  const tags: TagType[] = [
+    ...category.map(
+      (key: string): TagType => ({
+        type: SearchableTags.Category,
+        key,
+      }),
+    ),
+    ...organization.map(
+      (key: string): TagType => ({
+        type: SearchableTags.Organization,
+        key,
+      }),
+    ),
+    {
+      type: SearchableTags.Processentry,
+      key: 'true',
+    },
+  ]
+
+  const [
+    {
+      data: { searchResults },
+    },
+    {
+      data: { searchResults: countResults },
+    },
+    namespace,
+  ] = await Promise.all([
+    apolloClient.query<GetApplicationsQuery, QuerySearchResultsArgs>({
+      fetchPolicy: 'no-cache', // overriding because at least local caching is broken
```

Comment:
Did you try adding an environment variable to your .env.secret file which contains the bypass cache key, after I did that I've noticed that appending ?bypass-cache=<bypass-key> in the url locally does indeed refresh the cache

Reply:
This was remnants from copying over the Search screen as the template for this page. I think it can be safely removed. I'll run some local tests.